sources/packages/backend/apps/api/src/services/education-program/education-program.service.ts

```diff
@@ -202,6 +212,27 @@ export class EducationProgramService extends RecordDataModelService<EducationPro
     return this.repo.save(program);
   }
 
+  /**
+   * Ensures that when 'none of the above' is selected, other options are set to false.
+   * @throws CustomNamedError in case entrance requirements are in a wrong state.
+   * @param educationProgram education programs details to have entrance requirements validated.
+   */
+  private validateEntranceRequirements(educationProgram: SaveEducationProgram) {
```

Comment:
For my info, we are not planning on keeping this since the formio `dryRunSubmission` is going to validate it, right?

Reply:
yes, as the formio is already doing that validation we won't need to validate it again.